wdl-engine/src/inputs.rs

```diff
@@ -616,18 +646,50 @@ impl Inputs {
         let reader = BufReader::new(file);
         let object = mem::take(
             serde_json::from_reader::<_, JsonValue>(reader)
-                .with_context(|| {
-                    format!("failed to parse input file `{path}`", path = path.display())
-                })?
+                .context("failed to parse input file")?
```

Comment:
For this change to call `context` instead of `with_context`, we're losing the path to the file, which is always useful to have in errors.

Can we change this back to what we had with `with_context`?